Playbooks/playbook-Archer_Playbook.yml

```diff
@@ -0,0 +1,341 @@
+id: 8d19cd76-5546-4435-8d86-91baa5ae315c
```

Comment:
the location of the playbook is wrong, should be under TestPlaybooks folder, for instance look at:
https://github.com/demisto/content/blob/master/TestPlaybooks/playbook-urlscan.yml